src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -3256,6 +3257,58 @@
     ("SveVecReduceUnOpTest.template",     new Dictionary<string, string> { ["TestName"] = "Sve_OrAcross_uint",                                                                                     ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "OrAcross",                                                            ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt32",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "UInt32",                                                                                                                           ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetUInt32()",                                                                                                                                                      ["ValidateReduceOpResult"] = "Helpers.OrAcross(firstOp) != result[0]",                                                                                                              ["ValidateRemainingResults"] = "result[i] != 0"}),
     ("SveVecReduceUnOpTest.template",     new Dictionary<string, string> { ["TestName"] = "Sve_OrAcross_ulong",                                                                                    ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "OrAcross",                                                            ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt64",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "UInt64",                                                                                                                           ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetUInt64()",                                                                                                                                                      ["ValidateReduceOpResult"] = "Helpers.OrAcross(firstOp) != result[0]",                                                                                                              ["ValidateRemainingResults"] = "result[i] != 0"}),
 
+    ("ScalarImm2UnOpTest.template",       new Dictionary<string, string> {["TestName"] = "Sve_SaturatingDecrementBy16BitElementCount_int",                                                         ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "SaturatingDecrementBy16BitElementCount",                                                                ["RetBaseType"] = "Int32",                                    ["Op1BaseType"] = "Int32",                                   ["Op2BaseType"] = "Byte",                                    ["Op3BaseType"] = "SveMaskPattern", ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetInt32()",         ["Imm"] = "(Byte)2",  ["Imm2"] = "SveMaskPattern.All",                                    ["ValidateResult"] = "isUnexpectedResult = (result != (data - (2 * Unsafe.SizeOf<Vector<Int16>>() / sizeof(Int16))));",}),
```

Comment:
How would we do that? The program would error and exit.

Reply:
It will be similar to how we test APIs on unsupported platform in `RunUnsupportedScenario()`. Basically, the test method needs to catch the exception and make sure it is `ArgumentException` with the right message.